aip/0152.md

```diff
@@ -0,0 +1,84 @@
+---
+aip:
+  id: 152
+  state: reviewing
+  created: 2020-04-27
+permalink: /152
+redirect_from:
+  - /0152
+---
+
+# Jobs
+
+Occasionally, APIs may need to expose a task that takes significant time to
+complete, and where a transient [long-running operation][aip-151] is not
+appropriate. For example, a task could need to run repeatedly, or have separate
```

Comment:
```suggestion
appropriate. For example, a task could need to run on a recurring schedule, or have separate
```

Reply:
I would prefer focus on the _repeated_ aspect than the _scheduling_ aspect. (In theory, we have Cloud Scheduler for tasks that are run on a schedule, although that applies less elsewhere.)